### PR TITLE
Create DeleteRoleCommand and parser class

### LINKED ISSUE #54: Add deleteRole command class and parsing logic

### PR DESCRIPTION
**Pull request by BrendonLau**
Create deleteRole feature so that user can delete a role from a company.

1. DeleteRoleCommand class to handle the command and execution of deleting role.

2. DeleteRoleCommandParser class to handle input of user and to delete a role.

3.  Update test for 1. and 2. classes.

Main code for executing delete. 
image

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteRoleCommand.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import seedu.address.model.role.RoleManager;
+
+
+public class DeleteRoleCommand extends Command {
+    public static final String COMMAND_WORD = "deleteRole";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes the role identified by the index number used in the displayed company list \n"
+            + "followed by the index number used by the role in the specified company.\n"
+            + "Parameters: INDEX (must be a positive integer)\n"
```
Comment (justinyjt) [inline src/main/java/seedu/address/logic/commands/DeleteRoleCommand.java]: Would it be better to be more specific here with something like:

Parameters: COMPANY_INDEX ROLE_INDEX (both must be positive integers)

File: src/main/java/seedu/address/logic/commands/DeleteRoleCommand.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            + "Example: " + COMMAND_WORD + " 1 1";
+
+    public static final String MESSAGE_DELETE_ROLE_SUCCESS = "Deleted Role: %1$s  %1$s";
```
Comment (justinyjt) [inline src/main/java/seedu/address/logic/commands/DeleteRoleCommand.java]: Not sure if you intended to have double spaces between the indices for aesthetic purposes but just letting you know!

**Review by justinyjt — COMMENTED** [review]
LGTM overall!! Great job :D

File: src/main/java/seedu/address/logic/commands/DeleteRoleCommand.java (hunk 1 above)
Comment (BrendonLau) [reply src/main/java/seedu/address/logic/commands/DeleteRoleCommand.java]: Yup ok I changed the message.

File: src/main/java/seedu/address/logic/commands/DeleteRoleCommand.java (hunk 2 above)
Comment (BrendonLau) [reply src/main/java/seedu/address/logic/commands/DeleteRoleCommand.java]: Did not notice the double spacing. Fixed. Thanks.

**Pull request merged**